### DISCUSSION TITLE
Queued

### DISCUSSION THREAD

**Discussion by kagulati09** [vercel/community#539] [Help]
Hi, Its been at least 30 minutes and my deployment is queued. Is there something going on such as maintenance?

**Comment by swarnava** [comment]
Hey, I am not aware of any known issue. If you still facing this issue, could you create a case here: https://vercel.com/support/request ?

**Reply by toniengelhardt** [reply]
It seems to be not possible to create a case for build/deployment issues for free accounts...

**Reply by swarnava** [reply]
That's correct. Are you still experiencing this issue?

**Reply by toniengelhardt** [reply]
Yes, all my builds are stuck in queued since two days, even though no builds are running. I can also not transfer them to a Team account bc it says builds are running. Is there a way to force cancel all stuck builds?

**Reply by toniengelhardt** [reply]
User ID: hbLowZzQJBHbf00ooYal4jEy

One of the builds that's stuck: https://vercel.com/toniengelhardt/promptmetheus-website/G1YqZsqqSV6WUG1KMyXgzhLpSA3A

**Reply by amyegan** [reply]
Does it work if you [redeploy](https://vercel.com/docs/deployments/managing-deployments#redeploy-a-project) or if you [deploying with Vercel CLI](https://vercel.com/docs/cli/deploying-from-cli#) `vercel --prod`?

**Reply by toniengelhardt** [reply]
Thanks for looking at it Amy 🙏🏽

No, every time I try a new deploy it just gets queued, for all my apps. Also cancelling the currently queued ones doesn't help. It seems like there is a deploy stuck somewhere that is not visible anywhere...

**Reply by toniengelhardt** [reply]
UPDATE: not sure if you did something, but it's building again 🙏🏽
